test/e2e/lib/pages/media-page.js

```diff
@@ -24,7 +24,10 @@ export default class MediaPage extends AsyncBaseContainer {
 			this.driver,
 			By.css( '.media-library__list-item:not(.is-placeholder)' )
 		);
-		await driverHelper.clickWhenClickable( this.driver, By.css( '.media-library__list-item' ) );
+		await driverHelper.clickWhenClickable(
+			this.driver,
+			By.css( '.media-library__list-item.is-image' )
```

Comment:
As @Stojdza suggested ☝️:
```suggestion
			By.css( '.media-library__list-item .is-image' )
```